ui/pages/users/index.js

```diff
@@ -81,6 +81,7 @@ function Details({ user, admin, onDelete }) {
       title: 'Created',
       data: user?.created ? dayjs(user.created).fromNow() : '-',
     },
+    { title: 'Providers', data: user?.providerNames.join(', ') },
```

Comment:
nit:

You could consider 'Provided By' rather than 'Providers' to match the CLI

Reply:
That was what I had initially, but changed it to Providers for consistency in the UI